### ISSUE TITLE
Get and set regulating terminal for generator

### ISSUE THREAD

**Issue by vincentbarbesant** [powsybl/pypowsybl#207]
* **Do you want to request a *feature* or report a *bug*?**
A feature

* **What is the current behavior?**
The dataframe for generators does not return which terminal the voltage regulation of the generator is regulating
```
Data columns (total 12 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   energy_source         2768 non-null   object 
 1   target_p              2768 non-null   float64
 2   max_p                 2768 non-null   float64
 3   min_p                 2768 non-null   float64
 4   target_v              2768 non-null   float64
 5   target_q              2768 non-null   float64
 6   voltage_regulator_on  2768 non-null   bool   
 7   p                     2768 non-null   float64
 8   q                     1435 non-null   float64
 9   voltage_level_id      2768 non-null   object 
 10  bus_id                2768 non-null   object 
 11  genreCvg              2768 non-null   object 
```
* **What is the expected behavior?**
I want to be able to get and set the regulating terminal (possibly a remote one) for any generator

* **Please tell us about your environment:**
  - PowSyBl Version: 0.8.0
  - OS Version: Windows 10 Entreprise 64-bit (10.0, Build 18363) (18362.19h1_release.190318-1202)

**Comment by sylvlecl** [comment]
For the setter, we need to decide what will be the input : 

1.  any identifiable (generator, load, busbar) --> in that case the corresponding terminal will be used
2.  in bus breaker topology, this will not be very natural : we would like to be able to regulate a specific bus

However, 2. is a design issue in `powsybl-core`, not specific to pypowsybl, so we can go for 1. and handle it later.

**Comment by sylvlecl** [comment]
Implemented in #218 for node breaker networks only.

Issue #296 has been created for bus breaker implementation, and to discuss the corresponding design issues.
